### PR TITLE
[SIE-35] add Login page

### PR DESCRIPTION
**Pull request by alecherryy**

### CODE REVIEW

File: src/stories/pages/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+export const Login = ({ isDarkTheme }) => {
+  const [error, setError] = useState(false);
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const getUser = (e) => {
+    e.preventDefault();
+
+    const service = UserServices.getInstance();
+    console.log(service.getUsers()); // eslint-disable-line no-console
```
Comment (cmutuc25) [inline src/stories/pages/Login/Login.js:35]: Is there a reason for this? Shouldnt we not be using `console.log`

File: src/stories/pages/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    service.getUsers().then((users) => {
+      try {
+        users.forEach((item) => {
```
Comment (cmutuc25) [inline src/stories/pages/Login/Login.js:39]: Nitpicky but `find()` would make more sense to read. Also, shouldn't the user finding be handled in the backend?

**Review by cmutuc25 — APPROVED** [review]
I added some comments that are more questions and nitpicks than actual changes.

File: src/stories/pages/Login/Login.js (hunk 1 above)
Comment (alecherryy) [reply src/stories/pages/Login/Login.js:35]: I forgot it there lol. Thanks for catching it!

File: src/stories/pages/Login/Login.js (hunk 2 above)
Comment (alecherryy) [reply src/stories/pages/Login/Login.js:39]: @cmutuc25 there's a `findUserById` in the BE, however, we don't have access to that when a user first logs onto the platform.

Comment (alecherryy) [reply src/stories/pages/Login/Login.js:39]: Also, great suggestion! So much cleaner.

**Comment by alecherryy** [comment]
I merged master into this branch and re-tested the Login. All is good on my end.

**Pull request merged**